src/Features/CSharp/Portable/CodeFixes/HideBase/HideBaseCodeFixProvider.AddNewKeywordAction.cs

```diff
@@ -13,6 +15,12 @@ private class AddNewKeywordAction : CodeActions.CodeAction
             private readonly Document _document;
             private readonly SyntaxNode _node;
 
+            private static CSharpSyntaxFactsService SyntaxFacts => CSharpSyntaxFactsService.Instance;
+
+            private static Options.Option<CodeAnalysis.CodeStyle.CodeStyleOption<string>> PreferredModifierOrderOption => CodeStyle.CSharpCodeStyleOptions.PreferredModifierOrder;
+
+            private static OrderModifiers.CSharpOrderModifiersHelper OrderModifiersHelper => OrderModifiers.CSharpOrderModifiersHelper.Instance;
```

Comment:
💡 Consider inlining these properties and removing these lines since they are only used in the one method below

💡 Consider adding new `using` directives for the namespaces referenced by these types